index.js

```diff
@@ -318,7 +318,9 @@ Tree.prototype._join = function (data) {
     }
   }
 
+  this.overflowClassed(data.length)
```

Comment:
I'm running to the dentist. I'll try and think of something as I'm laying there in misery.